### ISSUE TITLE
The questions about mask in dlutils.py

### ISSUE THREAD

**Issue by ChenLiying555** [imperial-qore/TranAD#10]
There have two class, TransformerEncoderLayer and TransformerDecoderLayer.
The code as below:
```
 class TransformerEncoderLayer(nn.Module):
    def __init__(self, d_model, nhead, dim_feedforward=16, dropout=0):
        super(TransformerEncoderLayer, self).__init__()
        self.self_attn = nn.MultiheadAttention(d_model, nhead, dropout=dropout)
        self.linear1 = nn.Linear(d_model, dim_feedforward)
        self.dropout = nn.Dropout(dropout)
        self.linear2 = nn.Linear(dim_feedforward, d_model)
        self.dropout1 = nn.Dropout(dropout)
        self.dropout2 = nn.Dropout(dropout)'

        self.activation = nn.LeakyReLU(True)
    def forward(self, src,src_mask=None, src_key_padding_mask=None):
        src2 = self.self_attn(src, src, src)[0]
        src = src + self.dropout1(src2)
        src2 = self.linear2(self.dropout(self.activation(self.linear1(src))))
        src = src + self.dropout2(src2)
        return src
 ``` 

But the src_mask and src_key_padding_mask are equal to None, so does it means no mask during encoder? Aren't masks necessary? Why is it set to None?

Thank you

**Comment by shreshthtuli** [comment]
As noted in the readme, this is a refactored version for ease of use. We are working on creating another branch that includes the final models and tuned hyperparameters for all datasets used for result generation.
